recipes/entrypoints/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cp entrypoints.py $SP_DIR
```

Comment:
I just took a look at `flit` (and `flonda`) interesting concept! I like how you streamlined packaging. I am just not sure how to use that here.  @msarahan are you familiar with `flit`? Any ideas?